www/source/partials/docs/_commands-cli.html.md.erb

```diff
@@ -585,6 +585,7 @@ Builds a Plan using a Studio
 
 **FLAGS** 
 
+    -w, --windows    Use a Windows Studio instead of a Docker Studio (only available on Windows)
```

Comment:
This is good, although given our current process (which is to copy/paste CLI output when refreshing these docs), I wouldn't be surprised if these get left out the next time all commands get refreshed. Clearly we do need a better process here, but just making a note for the time-being.